compile/fullCompile/ratMainCompileScript.m

```diff
@@ -4,18 +4,24 @@
 % 
 % See also CODER, CODER.CONFIG, CODER.TYPEOF, CODEGEN.
 
+%% Deal with globals
+global verbose DEBUG
+DEBUG = 0;
+verbose = 1;
+
 %% Create configuration object of class 'coder.MexCodeConfig'.
 cfg = coder.config('mex');
 cfg.GenerateReport = true;
 cfg.EnableJIT = false;
 cfg.EnableOpenMP = true;
 cfg.TargetLang = 'C++';
+vfg.EnableVariableSizing = true;
```

Comment:
This line does not do anything because you are setting `vfg` instead of `cfg`, is this line still required?

Reply:
No - it can be removed (turns out it is true as default in cfg)